app/js/components/Loading.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+import style from './Loading.css';
+
+const Loading = ({ loaded }) => (
+    loaded ?
+        null :
+        <div className={style.wrapper}>
+            <div className={style.loaderBox}>
+                <div className={style.loader}>
+                </div>
+                <p className={style.label}>
+                    Connecting...
```

Comment:
I think the eventually could have a custom message, so as actions are sent it can update to a different state, like "Establishing connection with local server", "Fetching available apis", "Fetching plugins", etc... And also allow it to be used on any screen for any reason.